packages/@aws-cdk/aws-stepfunctions/README.md

```diff
@@ -538,6 +539,22 @@ const success = new stepfunctions.Fail(this, 'Fail', {
 });
 ```
 
+### Map
+
+A `Map` state can be used to run a set of steps for each element of an input array.
+A Map state will execute the same steps for multiple entries of an array in the state input.
+
+While the Parallel state executes multiple branches of steps using the same input, a Map state will
+execute the same steps for multiple entries of an array in the state input.
```

Comment:
Maybe put Map and Parallel in backquotes to keep it consistent.

Reply:
done